lib/vanagon/platform/windows.rb

```diff
@@ -107,7 +108,8 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/staging",
+        "mkdir -p $(tempdir)/{staging,wix}",
+        "#{@copy} -r wix/* $(tempdir)/wix",
```

Comment:
Wanna add an extra `/` after `$(tempdir)/wix` so we're extra clear we're moving stuff into a directory? It doesn't impact functionality at all, since this already does the right thing. It would just add extra clarity